hbase-common/src/main/java/org/apache/hadoop/hbase/PrivateCellUtil.java

```diff
@@ -2345,7 +2342,7 @@ public static void setTimestamp(Cell cell, long ts) throws IOException {
   }
 
   /**
-   * Sets the given timestamp to the cell. n * @param ts buffer containing the timestamp value
```

Comment:
I think that these comment lines starting with `n` are left-overs from the auto-formatting tool getting confused by newline characters.